argopy/plotters.py

```diff
@@ -45,6 +46,38 @@
                                                scale='50m',
                                                facecolor=[0.4,0.6,0.7])
 
+def open_dashboard(wmo=None, cyc=None, width="100%", height=1000, url=None, type='ea'):
+    """ Insert in a notebook the Euro-Argo dashboard page
+
+        Parameters
+        ----------
+        wmo: int
+            The float WMO to display. By default, this is set to None and will insert the general dashboard.
+
+        Returns
+        -------
+        IFrame: IPython.lib.display.IFrame
+    """
+    if type not in ['ea']:
+        raise InvalidDashboard("Invalid dashboard type")
+
+    from IPython.display import IFrame
+    if url is None:
+        if type=='ea': # Open Euro-Argo dashboard
+            if wmo is None:
+                url = "https://fleetmonitoring.euro-argo.eu"
+            else:
+                url = "https://fleetmonitoring.euro-argo.eu/float/{}".format(str(wmo))
+
+        ## argovis doesn't allow X-Frame insertion !
+        # elif type == 'argovis':
+        #     if cyc is None:
+        #         url = "https://argovis.colorado.edu/catalog/platforms/{}/page".format(str(wmo))
+        #     else:
+        #         url = "https://argovis.colorado.edu/catalog/profiles/{}_{}/page".format(str(wmo),str(cyc))
```

Comment:
why the "url" parameter ? Shouldn't "type" be enough ?
right now, this is possible : 
`argopy.dashboard(type="ea",url="http://wwz.ifremer.fr")`

Reply:
I foresee an internal usage within argopy, whereby we could trigger the function from a fetcher or another plotter functionality creating its own url.